libraries/Microsoft.Bot.Builder/CloudAdapterBase.cs

```diff
@@ -233,6 +233,28 @@ public override async Task CreateConversationAsync(string botAppId, string chann
             }
         }
 
+        /// <summary>
+        /// This is a helper to create the ClaimsIdentity structure from an appId that will be added to the TurnContext.
+        /// It is intended for use in proactive and named-pipe scenarios.
+        /// </summary>
+        /// <param name="botAppId">The bot's application id.</param>
+        /// <returns>A <see cref="ClaimsIdentity"/> with the audience and appId claims set to the appId.</returns>
+        protected static ClaimsIdentity CreateClaimsIdentity(string botAppId)
```

Comment:
Are these tests moved because a protected static should be before a protected method? and a private static should go before a private method?

Reply:
Yes, that is the order required by StyleCop